openmetadata-service/src/main/java/org/openmetadata/service/resources/dqtests/TestCaseResource.java

```diff
@@ -379,13 +380,19 @@ public ResultList<TestCase> listFromSearch(
     searchListFilter.addQueryParam("domain", domain);
     if (!nullOrEmpty(owner)) {
       EntityInterface entity;
+      StringBuffer owners = new StringBuffer();
```

Comment:
List would be better option no?

Reply:
We can, just wanted to keep the behavior consistent on the SearchList side of thing. LMK if that makes sense.